apps/pipeline/scripts/CLEANUP_OBSOLETE_SCRIPTS.sh

```
@@ -0,0 +1,164 @@
+echo ""
+echo "⚠️  주의: 이 스크립트는 신약+구약 분석이 완료된 후에만 실행하세요!"
+echo "⚠️  analyze-old-testament.ts가 실행 중이면 절대 실행하지 마세요!"
+echo ""
+read -p "계속하시겠습니까? (y/N): " confirm
+
+if [[ $confirm != [yY] ]]; then
+    echo "취소되었습니다."
+    exit 0
+fi
+
+cd /Users/peter/Dev/bible-transcription-site/apps/pipeline/scripts
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)